sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClientOptions.cs

```diff
@@ -25,6 +25,7 @@ public CommunicationIdentityClientOptions(ServiceVersion version = LatestVersion
         {
             ApiVersion = version switch
             {
+                ServiceVersion.V2021_02_22_preview1 => "2021-02-22-preview1",
```

Comment:
Should you make this the latest version? Line 17

Reply:
@minnieliu  - I was following Reza's feedback (https://github.com/Azure/azure-sdk-for-net/pull/19735#discussion_r605313145). It would make sense to do what you suggested if this was higher than the current LatestVersion. Please advise what's the best course of action, as there seems to be some conflicting feedback.

Reply:
After the API version "2021-02-22-preview1" has been increased as per the comment above, we should make this the latest version.

Reply:
This change is not needed anymore now, right?

Reply:
Yea, let's fix this later for the public preview release